docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+ - SASL username
+ - SASL domain
+ - SASL password
+
+To retrieve these values, use the following command in the terminal:
+
+.. code:: sh
+
+ ssh mon sudo cat /etc/postfix/sasl_passwd
+
+This will return a line like:
+
+.. code:: sh
+ 
+ [smtp.gmail.com]:587 testossec@gmail.com:AwfulPassword
+
+In this example, ``smtp.gmail.com`` is the SMTP server, ``587`` is the SMTP port,
+``testossec`` is the SASL username, ``gmail.com`` is the SASL domain, and 
+``AwfulPassword`` is the SASL password.
+
+(Optional) Retrieve HTTPS certificate files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If your *Source Interface* was configured to use HTTPS, you will need to copy
+three related files from the *Application Server* to the *Admin Workstation*.
+
+To retrieve these files, use the commands:
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop/install_files/ansible-base
+   ssh app sudo tar -c -C /var/lib ssl/  | tar xvf - 
+
+These commands will create a directory named 
+``~/Persistent/securedrop/install_files/ansible-base/ssl``
+on the *Admin Workstation*, containing your instance's SSL certificate, 
+certificate key, and chain file. When prompted for the names of these files
+during the next step, you should specify them relative to the 
+``install_files/ansible-base`` directory, i.e. as ``ssl/mydomain.crt``.
+
+Step 5: Configure and back up the application
+=============================================
+
+Next, configure the application using the files and info retrieved in the 
+previous steps. To do so, connect to the Tor network on the
+*Admin Workstation*, open a Terminal and run the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ ./securedrop-admin setup
+ ./securedrop-admin sdconfig
+
+.. note:: The ``./securedrop-admin setup`` command may take several minutes to complete, and may
+ fail due to network issues. If it fails, it's safe to run again.
+
+When prompted, fill in the values retrieved in the previous steps.
+
+Once complete, configure the *Admin Workstation* using the command:
+
+.. code:: sh
+
+  ./securedrop-admin tailsconfig
+
+This will set up desktop shortcuts for the *Source* and *Journalist Interfaces*,
+and configure adminstrative access to the servers.
+
+Once the ``./securedrop-admin tailsconfig`` command is complete, verify that you can connect to 
+the servers using ``ssh app`` and ``ssh mon``, accepting the host verification
+prompt if necessary.
+
+Next, back up the servers by running the following command in the terminal:
+
+.. code:: sh 
+ 
+ ./securedrop-admin backup
+
+
+Step 6: Run the ``./securedrop-admin install`` command
+======================================================
+
+After the ``./securedrop-admin backup`` command completes successfully, you should  undo the
+changes made to enable temporary local SSH access, by running the following 
+command:
+
+.. code:: sh
+
+ ./securedrop-admin install
+
+Once the command completes successfully, your new *Admin Workstation* is ready
+for use.
```

Comment:
At this point, I was still able to access app and mon server over the local network, even though SSH over Tor was configured. Restarting SSH resolved this, but that suggests to me that the `iptables` update was not in effect either. Perhaps the simplest step to recommend here is to reboot both servers?

Reply:
Good suggestion - the first nightly reboot will reset iptables and restart sshd, but adding in an explicit reboot isn't a big deal.